signals/generators/ios/template_methods.py

```diff
@@ -36,25 +35,51 @@ def method_parameters(api):
     # Create request object parameters
     request_object = get_api_request_object(api)
     if request_object:
-        parameters.extend(generate_field_parameters(request_object))
-        parameters.extend(generate_relationship_parameters(request_object))
+        parameters.extend(ObjCParameter.generate_field_parameters(request_object))
+        parameters.extend(ObjCParameter.generate_relationship_parameters(request_object))
 
     # Add id parameter if we need it
-    id_parameter = create_id_parameter(api.url_path, request_object)
+    id_parameter = ObjCParameter.create_id_parameter(api.url_path, request_object)
     if id_parameter:
         parameters.append(id_parameter)
 
     # Add required RestKit parameters
     parameters.extend([
-        Parameter(name="success",
-                  objc_type="void (^)(RKObjectRequestOperation *operation, RKMappingResult *mappingResult)"),
-        Parameter(name="failure",
-                  objc_type="void (^)(RKObjectRequestOperation *operation, NSError *error)")
+        ObjCParameter(name="success",
+                      objc_type="void (^)(RKObjectRequestOperation *operation, RKMappingResult *mappingResult)"),
+        ObjCParameter(name="failure",
+                      objc_type="void (^)(RKObjectRequestOperation *operation, NSError *error)")
     ])
 
     return create_parameter_signature(parameters)
 
 
+def swift_method_parameters(api):
```

Comment:
I'm thinking it might be cleaner to separate template methods into they're own sub files under the different templates?